manipulation/src/main/java/org/jboss/gm/manipulation/actions/ManifestUpdateAction.java

```diff
@@ -0,0 +1,40 @@
+package org.jboss.gm.manipulation.actions;
+
+import org.gradle.api.Action;
+import org.gradle.api.Project;
+import org.gradle.api.java.archives.internal.DefaultManifest;
+import org.gradle.api.plugins.osgi.OsgiManifest;
+import org.gradle.api.tasks.bundling.Jar;
+import org.jboss.gm.common.alignment.AlignmentModel;
+
+/**
+ * Overrides specified OSGI manifest entries.
+ * <p>
+ * Currently only overrides manifest of type OsgiManifest and only entries "Specification-Version" and
+ * "Implementation-Version".
+ */
+public class ManifestUpdateAction implements Action<Project> {
```

Comment:
Why do we need this?

Reply:
Does PME update the generated Manifests? If so, then I would assume we would want it here as well.

The only reason I can think of to have the manifests populated with this info is for `getClass().getPackage().getImplementationVersion` to return the updated version

Reply:
No PME doesn't because change the manifest explicitly, because generation uses the updated model.

Reply:
But ultimately the changes to the model are reflected in the manifest, correct? I think that is not the case for Gradle (although I'm not certain)

Reply:
Yes, changes to the model are reflected in the pom files, the manifests etc.

I am not sure with Gradle hence asking about this and the poms :-)

Reply:
Yes, for maven projects built in PNC the versions in the manifest are the updated versions (because projects are built with updated pom.xml). So this mimics maven project behavior.

I'm not sure if it's critical to have this, but it seems a good idea and it's not difficult. I thought that some customer automation could theoretically rely on this...

Reply:
👍